### ISSUE TITLE
Only show certain parameters for each content types supported by an endpoint

### ISSUE THREAD

**Issue by kalEl2001** [swaggo/swag#1146]
**Is your feature request related to a problem? Please describe.**
I currently have an endpoint that supports two kinds of content types, application/json and application/x-www-form-urlencoded. I would love to show only the fields required for each content type, but I haven't found any kind of way to do so.

**Describe the solution you'd like**
I would love to have an option to hide the fields I would not want to show when the user selected a specific type of parameter content type.

**Additional context**
// @Accept json
// @Param nameForJson body JsonDataType true "DescJson" => I want this to show only when application/json is selected
// @Accept x-www-form-urlencoded
// @Param nameForForm formData FormDataType true "DescForm" => I want this to show only when application/x-www-form-urlencoded is selected

Thank you for your attention and your incredible library.

**Comment by ubogdan** [comment]
I don't believe this is an actual swag issue, this seems to be related to [swagger-ui](https://github.com/swagger-api/swagger-ui).

**Comment by khudaibirdin** [comment]
@kalEl2001, @ubogdan  Is there any solution to this issue?
